### PR TITLE
Remove execfile_, use SourceFileLoader instead

### PR DESCRIPTION
**Pull request by c-bata**
We can use importlib because gunicorn drops Python2 support. In this PR, I remove `gunicorn/_compat.py` and use importlib.SourceFileLoader instead.

Notice: To import wsgi application, we've already used `importlib` (See https://github.com/benoitc/gunicorn/pull/2043 for details).

### CODE REVIEW

File: gunicorn/app/base.py [hunk 1]
```diff
@@ -93,22 +95,18 @@ def get_config_from_filename(self, filename):
...
+            loader = SourceFileLoader(module_name, filename)
+            loader.exec_module(mod)
         except Exception:
             print("Failed to read config file: %s" % filename, file=sys.stderr)
             traceback.print_exc()
             sys.stderr.flush()
             sys.exit(1)
 
-        return cfg
+        return mod.__dict__
```
Comment (berkerpeksag) [inline gunicorn/app/base.py]: Can't we use `vars(mod)` here?

File: gunicorn/app/base.py [hunk 2]
```diff
@@ -5,13 +5,15 @@
 import os
 import sys
 import traceback
+import types
+from importlib.machinery import SourceFileLoader
```
Comment (berkerpeksag) [inline gunicorn/app/base.py]: Nit: Can you import it as `import importlib.machinery` and alphabetize all imports?

**Comment by c-bata** [comment]
Thanks for reviewing. I've just pushed a commit to apply your feedbacks.

**Comment by berkerpeksag** [comment]
Thanks!

**Pull request merged**